nuxeo-features/nuxeo-thumbnail/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -13,5 +13,6 @@ Nuxeo-Component: OSGI-INF/thumbnail-commandline-imagemagick-contrib.xml,
  OSGI-INF/thumbnail-listener-contrib.xml,
  OSGI-INF/thumbnail-operations-contrib.xml,
  OSGI-INF/thumbnail-rendition-contrib.xml,
- OSGI-INF/marshallers-contrib.xml
+ OSGI-INF/marshallers-contrib.xml,
+ OSGI-INF/thumbnailing-bulk-contrib.xml
```

Comment:
Can you rename it to `thumbnail-bulk-contrib.xml` for consistency with other contribs?